### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Mygod** [realm01/DHCPv6-Client-Android#7]
```
05-12 02:39:17.596  1494  1494 D AndroidRuntime: Shutting down VM
05-12 02:39:17.597  1494  1494 E AndroidRuntime: FATAL EXCEPTION: main
05-12 02:39:17.597  1494  1494 E AndroidRuntime: Process: org.daduke.realmar.dhcpv6client, PID: 1494
05-12 02:39:17.597  1494  1494 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.View android.view.View.findViewById(int)' on a null object reference
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at org.daduke.realmar.dhcpv6client.MainFragment.get_ip_addresses(MainFragment.java:87)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at org.daduke.realmar.dhcpv6client.MainFragment$2.onReceive(MainFragment.java:78)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.support.v4.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:297)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.support.v4.content.LocalBroadcastManager.access$000(LocalBroadcastManager.java:46)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.support.v4.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:116)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6178)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:891)
05-12 02:39:17.597  1494  1494 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:781)
```

**Comment by realmar** [comment]
Thank you for your issue. I will have a look.

**Comment by realmar** [comment]
@Mygod should be fixed now in 1.5.5 (db6b261cb53a4cc61822a23273414a5811fafcec)

**Comment by Mygod** [comment]
Great! I'll see if it or something similar occurs again.
